modules/tide_content_collection/src/Plugin/Field/FieldWidget/ContentCollectionConfigurationWidgetEnhanced.php

```diff
@@ -0,0 +1,639 @@
+    $this->getIndex();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $plugin_id,
+      $plugin_definition,
+      $configuration['field_definition'],
+      $configuration['settings'],
+      $configuration['third_party_settings'],
+      $container->get('module_handler'),
+      $container->get('tide_automated_listing.sapi_index_helper')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function defaultSettings() {
+    return [
+      'schema_validation' => FALSE,
+      'callToAction' => FALSE,
+      'contentTypes' => TRUE,
+      'field_topic' => TRUE,
+      'field_tags' => TRUE,
+    ] + parent::defaultSettings();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsSummary() : array {
+    $summary = parent::settingsSummary();
+
+    $schema_validation = $this->getSetting('schema_validation');
+    $summary[] = $this->t('Validate against the JSON schema: @validation', [
+      '@validation' => $schema_validation ? $this->t('Yes') : $this->t('No'),
+    ]);
+
+    $callToAction = $this->getSetting('callToAction');
+    $summary[] = $this->t('Call To Action Field: @status', [
+      '@status' => $callToAction ? $this->t('Enabled') : $this->t('Disabled'),
+    ]);
+
+    $contentTypes = $this->getSetting('contentTypes');
+    $summary[] = $this->t('Content Types Field: @status', [
+      '@status' => $contentTypes ? $this->t('Enabled') : $this->t('Disabled'),
+    ]);
+
+    $field_topic = $this->getSetting('field_topic');
+    $summary[] = $this->t('Content Topics Field: @status', [
+      '@status' => $field_topic ? $this->t('Enabled') : $this->t('Disabled'),
+    ]);
+
+    $field_tags = $this->getSetting('field_tags');
+    $summary[] = $this->t('Content Tags Field: @status', [
+      '@status' => $field_tags ? $this->t('Enabled') : $this->t('Disabled'),
+    ]);
+
+    $entity_reference_fields = $this->getEntityReferenceFields();
+
+    if (!empty($entity_reference_fields)) {
+      foreach ($entity_reference_fields as $field_id => $field_label) {
+        $field_advanced = $this->getSetting($field_id);
+        $text = $this->t('Field: @status', [
+          '@status' => $field_advanced ? $this->t('Enabled') : $this->t('Disabled'),
+        ]);
+        $summary[] = $field_label['label'] . ' ' . $text;
+      }
+    }
+
+    return $summary;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsForm(array $form, FormStateInterface $form_state) : array {
+    $element = parent::settingsForm($form, $form_state);
+
+    $element['schema_validation'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Validate against the JSON schema'),
+      '#default_value' => $this->getSetting('schema_validation'),
+    ];
+    $element['callToAction'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enable/Disable Call To Action'),
```

Comment:
@joshua-salsadigital so is this checkbox to enable or disable?

Reply:
Added correct labels and changed layout to tabs.